docbot/action_config.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+type ActionConfig struct {
+	token *string
+	repo  *string
+	owner *string
+
+	labelPattern        *string
+	labelWatchSet       map[string]struct{}
+	labelMissing        *string
+	enableLabelMissing  *bool
+	enableLabelMultiple *bool
+}
+
+func NewActionConfig() (*ActionConfig, error) {
+	ownerRepoSlug := os.Getenv("GITHUB_REPOSITORY")
+	ownerRepo := strings.Split(ownerRepoSlug, "/")
+	if len(ownerRepo) != 2 {
+		return nil, fmt.Errorf("GITHUB_REPOSITORY is not found")
+	}
+	owner, repo := ownerRepo[0], ownerRepo[1]
+
+	token := os.Getenv("GITHUB_TOKEN")
+
+	labelPattern := os.Getenv("LABEL_PATTERN")
+	if len(labelPattern) == 0 {
+		labelPattern = "- \\[(.*?)\\] ?`(.+?)`"
+	}
+
+	labelWatchListSlug := os.Getenv("LABEL_WATCH_LIST")
+	labelWatchList := strings.Split(strings.TrimSpace(labelWatchListSlug), ",")
+	labelWatchSet := make(map[string]struct{})
+	for _, l := range labelWatchList {
+		labelWatchSet[l] = struct{}{}
```

Comment:
```suggestion
		if r := strings.TrimSpace(l); len(r) > 0 {
			labelWatchSet[r] = struct{}{}
		}
```

For `LABEL_WATCH_LIST == 'docs, docs-required,'`.